fundingmanager.go

```diff
@@ -492,7 +498,8 @@ func (f *fundingManager) handleFundingResponse(fmsg *fundingResponseMsg) {
 
 	// Register a new barrier for this channel to properly synchronize with
 	// the peer's readHandler once the channel is open.
-	fmsg.peer.barrierInits <- *outPoint
+	peer := f.findPeer(peerKey)
```

Comment:
I think this has been fixed in your follow-up PR, but just making a not here so we don't lose track: I don't think `findPeer` will be needed after all. Instead if we move the management of the barriers within the `fundingManager` we'd no longer need to be able to randomly look up a peer by it's public key.